.codesandbox/ci.json

```diff
@@ -18,5 +18,5 @@
     "packages/instantsearch-ui-components",
     "packages/instantsearch-core"
   ],
-  "node": "16"
+  "node": "18"
```

Comment:
This works now ! Node 18 will leave maintenance in April 2025 though

Reply:
finally! we can revive the vite/astro PRs too then